spot-ml/src/main/scala/org/apache/spot/netflow/FlowWordCreator.scala

```diff
@@ -71,55 +74,59 @@ class FlowWordCreator(timeCuts: Array[Double],
     */
   def flowWords(hour: Int, minute: Int, second: Int, srcPort: Int, dstPort: Int, ipkt: Long, ibyt: Long): FlowWords = {
 
+    Try {
+      val timeOfDay: Double = hour.toDouble + minute.toDouble / 60 + second.toDouble / 3600
 
-    val timeOfDay: Double = hour.toDouble + minute.toDouble / 60 + second.toDouble / 3600
+      val timeBin = Quantiles.bin(timeOfDay, timeCuts).toString()
+      val ibytBin = Quantiles.bin(ibyt, ibytCuts).toString()
+      val ipktBin = Quantiles.bin(ipkt, ipktCuts).toString()
```

Comment:
The change is just that the code got moved to a different line, I didn't add the .toString(). Are you suggesting to try without .toString()?